docs/user-guide/resources-properties.md

```diff
@@ -44,30 +44,42 @@ How to manage users and groups is a topic present in the [Managing Users](managi
+Only for map type resource, it is possible to add some details to the map. In this case the *Edit properties* window is the following:
 
 <img src="../img/resource-properties/edit-map-properties-panel_details.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
 With a click on the **Add new details** button <img src="../img/button/add_details_button.jpg" class="ms-docbutton"/> it opens a panel where the user can write the details of the map. 
 
 <img src="../img/resource-properties/details_panel.jpg" class="ms-docimage"/>
 
-The text can be edited and some links and images can be added through the [Text Editor Toolbar](text-editor-toolbar.md). Once the editing is done, the map details can be saved with the **Save** button <img src="../img/button/save_large_button.jpg" class="ms-docbutton"/> and other three buttons appear, in order to:
+The text can be edited and some links and images can be added through the [Text Editor Toolbar](text-editor-toolbar.md). Once the editing is done, the map details can be saved with the **Save** button <img src="../img/button/save_large_button.jpg" class="ms-docbutton"/> and other buttons appear on the *Edit properties* panel.
 
-* Show the details preview <img src="../img/button/details_preview_button.jpg" class="ms-docbutton"/> 
+<img src="../img/resource-properties/details_sheet_buttons.jpg" class="ms-docimage"/>
 
-* Edit the details <img src="../img/button/edit-details-button.jpg" class="ms-docbutton"/>
+From here, the user is allowed to:
 
-* Delete the details sheet <img src="../img/button/delete_white_button.jpg" class="ms-docbutton"/>
+* Show the details **preview** <img src="../img/button/details_preview_button.jpg" class="ms-docbutton"/> 
 
-Once the details are saved, the **Show details** button <img src="../img/button/details_button.jpg" class="ms-docbutton"/> appears on the map card in [Homepage](https://mapstore.geo-solutions.it/mapstore/#/)
+* **Edit** the details <img src="../img/button/edit-details-button.jpg" class="ms-docbutton"/>
+
+* Enable the **Show as modal** <img src="../img/button/show-as-modal-button.jpg" class="ms-docbutton"/> button, to show the details on a modal when the user clicks on <img src="../img/button/about_this_map.jpg" class="ms-docbutton"/> button, which is listed in the [Burger Menu](menu-bar.md#burger-menu) options
+
+<img src="../img/resource-properties/show-as-modal.jpg" class="ms-docimage"/>
+
+!!! note
+    If the **Show as modal** button is not activated once the user opens the *About this map* button, the details are displayed on a panel. <img src="../img/resource-properties/show-as-panel.jpg" class="ms-docimage"/>
+
+!!! warning
+    The *About this map* button is visible in the [Burger Menu](menu-bar.md#burger-menu) only when the details are present on the map.
+
+* Enable the **Show at startup** <img src="../img/button/show-at-starup-button.jpg" class="ms-docbutton"/> button: the user who opens the map will see the details displayed first
```

Comment:
* Enable the **Show at startup** <img src="../img/button/show-at-starup-button.jpg" class="ms-docbutton"/> button. If active, as soon as the user opens the map, the details panel is visualized.